### PR TITLE
Add `WhatsNewModal` Image Check

### PR DESCRIPTION
**Pull request by julieg18**
* Have the modal check if image exists before adding `GatsbyImage`. Otherwise a build will fail if a main blog image isn't provided

### CODE REVIEW

**Review by yathomasi — COMMENTED** [review]
Looks good 👍

**Review by yathomasi — APPROVED** [review]

File: src/components/Home/WhatsNewModal/index.tsx [hunk 1]
```diff
@@ -91,12 +91,14 @@ const WhatsNewModal: React.FC = () => {
...
-              />
+              {latestPost.picture && (
+                <GatsbyImage
+                  className={styles.image}
+                  alt=""
+                  objectFit="contain"
+                  image={latestPost.picture.gatsbyImageData}
+                />
+              )}
             </Link>
```
Comment (rogermparent) [inline src/components/Home/WhatsNewModal/index.tsx:102]: Should we put the conditional on the `Link` component so we don't render an empty link?

Comment (julieg18) [reply src/components/Home/WhatsNewModal/index.tsx:102]: Nice catch! I'll update the code!

**Review by rogermparent — APPROVED** [review]
Looks good!

**Pull request merged**